inappmessaging/src/main/java/com/rakuten/tech/mobile/inappmessaging/runtime/service/DisplayMessageJobIntentService.kt

```diff
@@ -63,13 +63,16 @@ internal class DisplayMessageJobIntentService : JobIntentService() {
         hostActivity: Activity,
         imageUrl: String
     ) {
-        // If Fresco has not been initialized, initialize it first.
-        Fresco.getImagePipeline()
-            .prefetchToBitmapCache(ImageRequest.fromUri(imageUrl), null /* callerContext */)
-            .subscribe(
-                ImagePrefetchSubscriber(message, hostActivity),
-                UiThreadImmediateExecutorService.getInstance()
-            )
+        Picasso.get().load(imageUrl).fetch(object : Callback {
+            override fun onSuccess() {
+                displayMessage(message, hostActivity)
+                Timber.tag(TAG).d("Download image completed")
+            }
+
+            override fun onError(e: Exception?) {
+                Timber.tag(TAG).e(e, "Download image failed")
```

Comment:
Do we have a test that checks if message is not displayed in case of image download error?

Reply:
Picasso keeps a weak reference to the Callback instance and will be garbage collected if not keep a strong reference to it, I added it as a companion object to avoid GCing it. I created a separate ticket to handle the UT (SDKCF-4572).
 The manual test is working, and the message will not be displayed.

Reply:
@leila-gu 
make sure to check if it is possible to remove the strong reference on the other ticket.

Reply:
Done